crates/ruff_linter/src/rules/ruff/rules/logging_eager_conversion.rs

```diff
@@ -0,0 +1,137 @@
+use ruff_macros::{ViolationMetadata, derive_message_formats};
+use ruff_python_ast::{self as ast, Expr};
+use std::str::FromStr;
+
+use crate::rules::flake8_logging_format::rules::{LoggingCallType, find_logging_call};
+use ruff_python_literal::cformat::{CFormatPart, CFormatString, CFormatType};
+use ruff_python_literal::format::FormatConversion;
+use ruff_text_size::Ranged;
+
+use crate::Violation;
+use crate::checkers::ast::Checker;
+use crate::registry::Rule;
```

Comment:
I wasn't too worried about this, but since I found a few other nits, would you mind sorting these into `std`, "third-party," and first-party blocks:


```suggestion
use std::str::FromStr;

use ruff_macros::{ViolationMetadata, derive_message_formats};
use ruff_python_ast::{self as ast, Expr};
use ruff_python_literal::cformat::{CFormatPart, CFormatString, CFormatType};
use ruff_python_literal::format::FormatConversion;
use ruff_text_size::Ranged;

use crate::Violation;
use crate::checkers::ast::Checker;
use crate::registry::Rule;
use crate::rules::flake8_logging_format::rules::{LoggingCallType, find_logging_call};
```

Unfortunately this still isn't supported by stable rustfmt, so I try to do it manually if I notice it.